sdk/search/generate.ps1

```diff
@@ -7,13 +7,13 @@ Push-Location $PSScriptRoot/Azure.Search.Documents/src/
 # because AutoRest doesn't play well with two remote swagger files.  (We only
 # warn on failure so offline regeneration plays nicely for those of us coding
 # on the bus.)
-Write-Output "Copying latest swagger files..."
-Invoke-WebRequest https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchindex.json `
-    -OutFile ./swagger/searchindex.json `
-    -ErrorAction Continue
-Invoke-WebRequest https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchservice.json `
-    -OutFile ./swagger/searchservice.json `
-    -ErrorAction Continue
+#Write-Output "Copying latest swagger files..."
+#Invoke-WebRequest https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchindex.json `
+#    -OutFile ./swagger/searchindex.json `
+#    -ErrorAction Continue
+#Invoke-WebRequest https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchservice.json `
+#    -OutFile ./swagger/searchservice.json `
+#    -ErrorAction Continue
```

Comment:
Why are we commenting this out?  We didn't modify the swagger files locally by hand?

Reply:
I meant to revert that. I had to do this in steps because of the model renames causing errors during generation. Ultimately, I reverted my changes as I went but forgot that the first commit included this (phase one of the move/renames - I couldn't also take the new version).